src/types/index.ts

```diff
@@ -219,12 +219,12 @@ export type FilterPageCopy = StrapiBaseEntry & {
 }
 
 export type CoursePageCopy = StrapiBaseEntry & {
-  HeaderCourseContent: string
-  Authors: string
-  DownloadCourseContent: string
-  PowerpointDownloadDescription: string
-  DocxDownloadDescription: string
-  IntermediateHeader: string
-  CourseCreatedAt: string
-  CourseUpdatedAt: string
+  descriptionHeader: string
```

Comment:
Do these really have their first letter in lowercase? If so, we should change it since every other field in Strapi have their names capitalized.

Reply:
I will fix that